themes/theme-ios11/pages/Product/components/Content/index.jsx

```diff
@@ -4,15 +4,35 @@ import { Conditioner } from '@shopgate/pwa-core';
 import TaxDisclaimer from '@shopgate/pwa-ui-shared/TaxDisclaimer';
 import { Section } from '@shopgate/engage/a11y';
 import {
-  ProductProperties, RelationsSlider, Description, ProductContext, Options, Characteristics,
+  ProductProperties,
+  RelationsSlider,
+  Description,
+  ProductContext,
+  Options,
+  Characteristics,
+  ProductUnitQuantityPicker,
+  OrderQuantityHint,
 } from '@shopgate/engage/product';
 import { Reviews } from '@shopgate/engage/reviews';
+import { css } from 'glamor';
 import Media from '../Media';
 import Header from '../Header';
 import AppBar from '../AppBar';
 import AddToCartBar from '../AddToCartBar';
 import connect from './connector';
 
+const styles = {
```

Comment:
styles for the quantity picker shouldn't be inside this file here. should be moved to specific component